commit/trk2dictionary/trk2dictionary.pyx

```diff
@@ -250,13 +250,14 @@ cpdef run( filename_tractogram=None, path_out=None, filename_peaks=None, filenam
     # Streamlines from tractogram
     print( '\t- Tractogram' )
     
+    if not exists(filename_tractogram):
+        ERROR( 'Tractogram file not found: %s' % filename_tractogram )        
```

Comment:
Could you add also a line in the CHANGELOG to keep track that we "fixed" this?

Reply:
Yes